### PR TITLE
Remove usage of arrow from parser RU

### PR DESCRIPTION
**Pull request by albertoursino**
## Issue

Related issue: #6135.

## Description

Removed arrow usage from parser [RU](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/parsers/RU.py).

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser RU`, `poetry run test_parser RU-1`, `poetry run test_parser RU-2`, `poetry run test_parser RU-AS`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: parsers/RU.py [hunk 1]
```diff
@@ -115,13 +115,13 @@ def fetch_production_1st_synchronous_zone(
         raise NotImplementedError("This parser is not able to parse given zone")
 
     if target_datetime:
-        target_datetime_tz = arrow.get(target_datetime).to(tz)
+        target_datetime_tz = target_datetime.replace(tzinfo=TIMEZONE)
```
Comment (VIKTORVAV99) [inline parsers/RU.py:118]: The equivelant of arrows `.to()` is `.astimezone()`
```suggestion
        target_datetime_tz = target_datetime.astimezone(tz=TIMEZONE)
```

File: parsers/RU.py [hunk 2]
```diff
@@ -181,12 +182,12 @@ def fetch_production_2nd_synchronous_zone(
...
```
Comment (VIKTORVAV99) [inline parsers/RU.py:185]:
```suggestion
        target_datetime_tz = target_datetime.astimezone(tz=TIMEZONE)
```
same here

File: parsers/RU.py [hunk 3]
```diff
@@ -266,12 +268,9 @@ def fetch_exchange(
...
 ) -> list:
     """Requests the last known power exchange (in MW) between two zones."""
-    if target_datetime:
-        today = arrow.get(target_datetime, "YYYYMMDD")
-    else:
-        today = arrow.utcnow()
+    today = target_datetime if target_datetime else datetime.now(timezone.utc)
 
-    date = today.format("YYYY-MM-DD")
+    date = today.strftime("%Y-%m-%d")
```
Comment (VIKTORVAV99) [inline parsers/RU.py:273]:
```suggestion
    date = today.date().isoformat()
```
Slightly cleaner in my opinion.

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Some general comments and only 2 things that need changing really, but nice work!

**Comment by albertoursino** [comment]
Thank you for your review @VIKTORVAV99. I've made the changes according to your suggestions.

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**